docs/arch/0000-Inventory.rst

```diff
@@ -0,0 +1,63 @@
+3. Inventory
+============
+
+Date: 2021-08-24
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+This document defines the inventory. The inventory is a list of items and
+actions. Their relationships are defined as well as usage constraints.
+
+Think about how you put together ideas in your head. You connect concepts
+knowing resources and processes. For example, how do you make coffee? You know a
+coffee machine gives you brewed coffee. (Need machine, grinder, store sells
+coffee, other store sells machine, delivery, etc. or coffee shop).
+
```

Comment:
Trying to come up with a standard format for:
1. **Pieces of information, assets, resources, entities, processes (operations, operation implementations).
2. Connections between those pieces
3. Considerations which need to be taken into account when
  a. The piece is used in a flow
  b. The piece is used in relation to other specific pieces
  c. Flucuating circumstances applied at flow execution

Reply:
What do we do in our heads when we're learning? We're trying to get information into a form in our heads that when presented with a problem allows us to connect the dots between the things we know so as to use the most relevant information to tackle the problem at hand in the most effective way possible.

information as it currently exists and is expressed does not capture connections to other pieces of information in a meaningful way so as to enable the effective use of relevant information when solving for a task at had.

Reply:
POC to prod model

- Prototype
- Refactor into operations

Reply:
https://github.blog/2021-12-09-introducing-stack-graphs/

Reply:
Related manifests and schemas (and existing public schema inventories linked): https://github.com/intel/dffml/issues/1279#issuecomment-1013468346

Reply:
This looks possibly applicable to us https://www.w3.org/TR/did-core/

Reply:
DID stuff could be used to facilitate mappings between the same resource as it is identified in different systems, for example. PyPi package and it's source code repo

Reply:
https://ortelius.io/

Reply:
https://github.com/opencontainers/distribution-spec/blob/main/spec.md#enabling-the-referrers-api

Reply:
https://omeka.org/s/docs/user-manual/content/resource-template/

Reply:
https://github.com/decentralized-identity/ion
Identity Overlay Network (ION) will help us with being a out of the box id solution

Reply:
https://twitter.com/OR13b/status/1595804006596882434

Reply:
https://github.com/intel/dffml/discussions/1406#discussioncomment-3829553

Reply:
https://github.com/cayleygraph/cayley

Reply:
Hmmmm, if we use  thc.org/segfault and throw compute on devcloud under it that could be cool too

Reply:
Thank you Steve: https://deepai.org/machine-learning-glossary-and-terms/association-learning